### ISSUE TITLE
Als consumer wil ik informatieobjecten via multipart kunnen uploaden in plaats van base64

### ISSUE THREAD

**Issue by sergei-maertens** [VNG-Realisatie/gemma-zaken#983]
...zodat ik performantere mechanismes kan gebruiken.

Base64 heeft worst-case een opblaasfactor van 33%, maar verder ook restricties dat binary data in memory geladen moet worden om base64 te encoden, wat het vervelend maakt voor clients/consumers.

In Open API 3 land is er sprake van multipart bodies waarbij binary data direct verstuurd wordt, samen met een JSON body, wat merkelijk performanter is. Dit heeft een eigen `Content-Type` header, en kan dus naast de base64 oplossing bestaan, waardoor consumers flexibiliteit hebben.

Onderstaande "vink-lijst" is voor het team zelf om in te vullen en hoeft niet door de indiender te worden ingevuld.

**Bepaling prioriteit door PO**

- [ ] verbreding of verdieping API's
- [ ] stimuleert gebruik door gemeenten
- [ ] stimuleert gebruik door leveranciers

... eventueel nog toelichting door PO

**Definition of ready**
- [ ] Iedereen in het team begrijpt de user story
- [ ] de gewenste (aanvulling op de) functionaliteit van de API's duidelijk en beschreven is.
- [ ] Is klein genoeg (maximaal 1/5 van sprint)
- [ ] Product Owner akkoord en voorzien van prioriteit (mag alleen afgevinkt worden door PO)
- [ ] Idee hebben van hoe deze user story kan worden gedemonstreerd.
- [ ] _Globale_ oplossingsrichting bekend
- [ ] Vastgelegd in Github en geplaatst in kolom ready

**Definition of done**
- [ ] Er is een OAS 3.0 specificatie
- [ ] Er is een referentieimplementatie
- [ ] Er zijn tests(cases) aanwezig die de wijziging aantonen en waarmee de user story getest kan worden.
- [ ] De technische specificatie (standaard.md) is gepubliceerd leesbaar
- [ ] Gebruikte gegevensmodel is na **iedere iedere sprint** bijgewerkt.

**Acceptatiecriteria**
- [ ] De DSO URI- en API-strategie worden gevolgd of afwijkingen zijn vastgelegd als ontwerp keuze
- [ ] Er zijn geen bekende GEMMA tegenstrijdigheden **of** afwijkingen zijn vastgelegd.

**Taken**
- [ ] Implementeren in referentie-implementatie [_verantwoordelijke_]
- [ ] Schrijven (unit) test voor referentie-implementatie [_verantwoordelijke_]
- [ ] Genereren/opstellen van OAS 3.0 [_verantwoordelijke_]
- [ ] Human Readable publiceren Open API Specificatie (v.3.0) [_verantwoordelijke_]
- [ ] Documentatie bijwerken
- [ ] Gegevensmodel bijwerken

**Comment by Hugo-ter-Doest** [comment]
@sergei-maertens is deze dubbel met #1078?

**Comment by sergei-maertens** [comment]
Nee, dit is een alternatief voor base64, maar lost niet de oorzaak op van bestanden-groter-dan-3GB. Het is een alternatief op puur `application/json` Content-Type en laat je toe om binary data in de reqest op te nemen.

Met de bestaande oplossingen en het werk in #1078 zou ik deze gewoon prio L maken.

**Comment by michielverhoef** [comment]
noob-alert dus als ik het goed begrijp kan een consumer dan zelf aangeven op welke manier binary data meegestuurd/opgevraagd moet worden?  Meesturen lijkt me vrij simpel, met opvragen kun je dan aangeven in de requestheader hoe het antwoord geencodeerd moet worden?
